tests-ng/test_dmesg.py

```diff
@@ -0,0 +1,20 @@
+import re
+from pathlib import Path
+
+import pytest
+
+
+@pytest.mark.feature("server and not metal", reason="needs server feature")
+def test_dmesg_sysctl_config():
+    config_file = "/etc/sysctl.d/40-allow-nonroot-dmesg.conf"
+    assert Path(config_file).exists()
+    assert re.search(
+        r"(?m)^\s*(?!#)kernel\.dmesg_restrict\s*=\s*0", Path(config_file).read_text()
+    )
+
+
+@pytest.mark.feature(
+    "server and booted and not metal", reason="sysctl needs a booted system"
```

Comment:
- `booted` is not a feature, use https://github.com/gardenlinux/gardenlinux/blob/main/tests-ng/DEVELOPER.md#pytestmarkbootedreason.